osquery/tables/system/posix/ulimit_info.cpp

```diff
@@ -0,0 +1,68 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <climits>
+#include <map>
+#include <string>
+
+#include <sys/resource.h>
+
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+namespace osquery {
+namespace tables {
+
+void getLimit(QueryData& results) {
+  std::map<std::string, int> resource_map;
```

Comment:
Rather than assigning all of the values after declaring the variable, let's initialize this:
```
std::map<std::string, int> resource_map = {
    { "cpu", RLIMIT_CPU},
    { "fsize", RLIMIT_FSIZE},
    { "data", RLIMIT_DATA},
    { "stack", RLIMIT_STACK},
    { "core", RLIMIT_CORE},
    { "rss", RLIMIT_RSS},
    { "memlock", RLIMIT_MEMLOCK},
    { "nproc", RLIMIT_NPROC},
    { "nofile", RLIMIT_NOFILE},
    { "as", RLIMIT_AS},
    { "locks", RLIMIT_LOCKS},
    { "sigpending", RLIMIT_SIGPENDING},
    { "msgqueue", RLIMIT_MSGQUEUE},
    { "nice", RLIMIT_NICE},
    { "rtprio", RLIMIT_RTPRIO},
};
```